src/dvclive/plots/image.py

```diff
@@ -19,6 +19,8 @@ def could_log(val: object) -> bool:
             return True
         if val.__class__.__module__ == "numpy":
             return True
+        if isinstance(val, PurePath) or isinstance(val, str):
+            return True
```

Comment:
Should we check if the path exists and has a supported extension? Or is this intentionally left to later when we try to read it as an image?

Do we want to support a directory of images here? Not a blocker, just another idea to mention.

Reply:
I think it's fine to fail when it tries to read it.

> has a supported extension

Good question. Let me think and see if we want to check it here. Not sure. E.g. we don't check in the `if val.__class__.__module__ == "PIL.Image"` that is has the right format?

> Do we want to support a directory of images here?

Yes, can be a follow up later if this is needed.

Reply:
> we don't check in the if val.__class__.__module__ == "PIL.Image" that is has the right format?

I think is ok to let PIL handle the checks. Their errors are already explicit